ssl/record/ssl3_record.c

```diff
@@ -1156,6 +1158,26 @@ int tls1_enc(SSL *s, SSL3_RECORD *recs, size_t n_recs, int sending)
             }
         }
 
+        if (!SSL_IS_DTLS(s) && tlstree_enc) {
+            unsigned char *seq;
+            int decrement_seq = 0;
+            /*
```

Comment:
I think an empty line would be nice here

Reply:
Fixed. Thanks!